terraform/module-ses/redis.tf

```diff
@@ -1,50 +1,51 @@
 resource "aws_security_group" "redis" {
-  count       = "${var.create_elasticache == "true" ? 1 : 0}"
+  count       = var.create_elasticache == "true" ? 1 : 0
```

Comment:
We are now able to use bool directly

we should change the create_* to `count = var.create_elasticache ? 1 : 0`
and ensure that the sample variable mention boot ` #. create_elasticache (optional, bool): true`